6_Model Comparison.ipynb

```diff
@@ -0,0 +1,6627 @@
+{
```

Comment:
<p>I would say "standard deviation" instead of "error", because stats people might assume you're talking about an error term and not precision about a distribution's mean.</p>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Changed